scripts/dependencies/serverless/serverlessSub.yaml

```diff
@@ -7,10 +7,10 @@ spec:
   channel: stable
   name: serverless-operator
   # consume from specific catalog source for the specific version of serverless midstream
-  source: serverless-operator-v1-24-0
+  source: serverless-operator-v1-25-0
   sourceNamespace: openshift-marketplace
   installPlanApproval: Manual
-  startingCSV: serverless-operator.v1.24.0
+  startingCSV: serverless-operator.v1.25.0
```

Comment:
These scripts are meant to be run manually. AFAIK it is not possible to install 1.24 because it is already EOL. In order to test these changes on an OCP cluster I had to update these installation scripts with the current situation we have on dev

Reply:
OK, so it sounds like the best thing for us to do is take a maintenance window in `prod` in the new year and:

1. Upgrade to Serverless 1.25
2. Apply these Istio changes

So that we're consistent with the version of Serverless across all environments. Make sense to you?

Reply:
yep sure thing